contracts/DifficultyOracle.sol

```diff
@@ -2,13 +2,24 @@ pragma solidity ^0.5.1;
 import "@gnosis.pm/hg-contracts/contracts/PredictionMarketSystem.sol";
 
 contract DifficultyOracle {
+    /// Prediction market system this oracle reports to.
     PredictionMarketSystem public pmSystem;
+    /// Beginning of time window in which valid reports may be generated. Value is inclusive, meaning that the first valid timestamp is this.
     uint public startTime;
+    /// End of time window in which valid reports may be generated. Value is inclusive, meaning that the last valid timestamp is this.
     uint public endTime;
+    /// Difficulty level at which oracle would report a full value in the first slot if the actual difficulty was less than or equal to this target, and a full value in the second slot if the actual difficulty exceeded the target.
     uint public difficultyTarget;
+    /// Question ID oracle will use during report to the prediction market system. (AUDIT: this has no accessor unlike the other variables?)
     bytes32 questionId;
-    
+
+    /// @dev Emitted upon the successful reporting of whether the difficulty has exceeded the difficulty target to the prediction market system.
+    /// @param startTime Beginning of time window in which valid reports may be generated. (AUDIT: duplicate of storage var)
```

Comment:
What do you mean by duplicate of storage var? That the event parameter name is identical to a global variable and should be underscored, is that a problem?

Reply:
I don't see why this is an issue here?